### ISSUE TITLE
Client Token is missing in the Facebook Developer Console - how can I upgrade?

### ISSUE THREAD

**Issue by henieek** [facebook/facebook-android-sdk#1134]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [Github issues](https://github.com/facebook/facebook-android-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Java version

n/a

### Android version

n/a

### Android SDK version

n/a

### Installation platform & version

n/a

### Package

Other / I don't know

### Goals

I'm really sorry for raising this as a bug - it's not really a bug, but rather a question. I tried to raise it on the Developer Community Forum, but we didn't get any answer. (https://developers.facebook.com/community/threads/510098921130141/?post_id=510098924463474)

Our app is currently running the Facebook Android SDK v12.3.0, which is pretty outdated. Recently we wanted to proceed with an update, but encountered an unexpected problem. The new SDK requires a Client Token which our application doesn't seem to have:

![image](https://user-images.githubusercontent.com/336158/203324281-300b7f97-34a4-4dd1-9a2b-94162fa31aaa.png)

I suppose we can generate the Client Token by clicking "Reset" - our worry however is that this might break existing clients who are still running an outdated version of Facebook SDK, thus naturally don't supply the Client Token to any requests.

Could anyone please confirm whether the "Reset" action is safe and backwards-compatible? Our usage is limited to Login with Facebook (via LoginManager) and Friends Request (GraphRequest.newMyFriendsRequest).

### Expected results

Client Token is available for our app

### Actual results

Client Token is not available for our app

### Code samples & details

```java
// INSERT YOUR CODE HERE
```

**Comment by eloisance** [comment]
Hello @henieek ,

We did it few days ago in our company.

It was the same story for us, no client token in the dashboard but was needed to upgrade the SDK.

We did a ticket to FB support to ask the same question as you and they told us it is safe as long as we are not using any ClientToken yet for any production users.

So we did generate the token, everything was fine for our apps on prod (web, iOS and Android) and we proceed with the upgrade of the SDK now.

Hope it can help, just a feedback.
